reya_data_feed/resources/wallet.py

```diff
@@ -0,0 +1,254 @@
+    """Resource for accessing wallet orders."""
+    
+    def __init__(self, socket: 'ReyaSocket'):
+        """Initialize the wallet orders resource.
+        
+        Args:
+            socket: The WebSocket connection to use for this resource.
+        """
+        super().__init__(socket, "/api/trading/wallet/{address}/orders")
+    
+    def for_wallet(self, address: str) -> 'WalletOrdersSubscription':
+        """Create a subscription for a specific wallet's orders.
+        
+        Args:
+            address: The wallet address.
+            
+        Returns:
+            A subscription object for the wallet orders.
+        """
+        return WalletOrdersSubscription(self.socket, address)
+
+class WalletBalancesResource(SubscribableParameterizedResource):
+    """Resource for accessing wallet account balances."""
+    
+    def __init__(self, socket: 'ReyaSocket'):
+        """Initialize the wallet balances resource.
+        
+        Args:
+            socket: The WebSocket connection to use for this resource.
+        """
+        super().__init__(socket, "/api/trading/wallet/{address}/accounts/balances")
+    
+    def for_wallet(self, address: str) -> 'WalletBalancesSubscription':
+        """Create a subscription for a specific wallet's account balances.
+        
+        Args:
+            address: The wallet address.
+            
+        Returns:
+            A subscription object for the wallet balances.
+        """
+        return WalletBalancesSubscription(self.socket, address)
+
+class WalletPositionsSubscription:
+    """Manages a subscription to positions for a specific wallet."""
+    
+    def __init__(self, socket: 'ReyaSocket', address: str):
+        """Initialize a wallet positions subscription.
+        
+        Args:
+            socket: The WebSocket connection to use for this subscription.
+            address: The wallet address.
+        """
+        self.socket = socket
+        self.address = address
+        self.path = f"/api/trading/wallet/{address}/positions"
+    
+    def subscribe(self, batched: bool = False) -> None:
+        """Subscribe to wallet positions.
+        
+        Args:
+            batched: Whether to receive updates in batches.
+        """
+        self.socket.send_subscribe(channel=self.path, batched=batched)
+    
+    def unsubscribe(self) -> None:
+        """Unsubscribe from wallet positions."""
+        self.socket.send_unsubscribe(channel=self.path)
+
+class WalletOrdersSubscription:
+    """Manages a subscription to orders for a specific wallet."""
+    
+    def __init__(self, socket: 'ReyaSocket', address: str):
+        """Initialize a wallet orders subscription.
+        
+        Args:
+            socket: The WebSocket connection to use for this subscription.
+            address: The wallet address.
+        """
+        self.socket = socket
+        self.address = address
+        self.path = f"/api/trading/wallet/{address}/orders"
```

Comment:
quite confident that the /orders endpoint is not currently used in production in our ui and unlikely to be used by other clients (but i think we should be able to double check this)

i believe we should be able to rename orders to trades